.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/no-commonjs */
+module.exports = {
+    extends: ["@khanacademy"],
+    parser: "@babel/eslint-parser",
+    parserOptions: {
+        babelOptions: {
+            configFile: "./config/build/babel.config.js",
+        },
+    },
+    plugins: ["@babel", "import", "jest", "promise", "monorepo", "disable"],
+    settings: {
+        "eslint-plugin-disable": {
+            paths: {
+                react: ["./*.js", "src/*.js"],
+                "jsx-a11y": ["./*.js", "src/*.js"],
+            },
+        },
+    },
+    globals: {
+        __IS_BROWSER__: "readonly",
+    },
+    overrides: [
+        {
+            files: ["utils/*.js"],
+            rules: {
+                "import/no-commonjs": "off",
+            },
+        },
+        {
+            files: ["**/__tests__/**/*.test.js"],
```

Comment:
Do we expect to have sub-directories in `__tests__` dirs?  Not sure it matters... just curious.

Reply:
That's what we've been doing for other non-webapp repos.